cmd/utils/flags.go

```diff
@@ -227,9 +227,9 @@ var (
 		Name:  "whitelist",
 		Usage: "Comma separated block number-to-hash mappings to enforce (<number>=<hash>)",
 	}
-	EtherbaseFlag = cli.StringFlag{
-		Name:  "etherbase",
-		Usage: "Public address for transaction broadcasting and block mining rewards (default = first account)",
+	TxFeeRecipientFlag = cli.StringFlag{
```

Comment:
@YazzyYaz @pranaymohan we are changing (and splitting )`--etherbase` to `--tx-fee-recipient` and `--miner.validator`

1. etherbase dissapears => change to the docs? maybe just leave it to return an erro msg?
2. i'm tempted to remove this: deafult = first account behaviour. Doesn't seem to be one of those things you want to "accidentally set". What do you think?

Reply:
1. Let's change in docs for sure. Not sure I understand what you mean by error message? Another option is we can have etherbase be syntatic sugar for "I want both tx-fee and miner-fee to go to the same address" -- this way it won't break anyone who doesn't care. Also why do we keep the prefix `Miner` -- should we refactor to something else at some point, to get away from connotations of PoW?
2. +1, especially since it may lead to funds being sent elsewhere, it should probably fail if not set properly.